examples/java/core/enumerations/src/main/bond/enumerations.bond

```diff
@@ -0,0 +1,21 @@
+namespace Examples;
+
+enum Color
+{
+    Orange = 1,
+    Yellow = 2,
+}
+
+enum Fruit
+{
+    Apple = 1,
+    Orange = 2,
+}
+
+enum Limits
+{
+    Int32Min = -2147483648,
+    Int32Max = 2147483647,
+    UInt32Min = 0,
+    UInt32Max = 0xFFFFFFFF,
```

Comment:
Perhaps we shouldn't port it exactly as-is. Rather than copy the uint stuff, which isn't really meaningful in Java world, it would be more useful to define an enum with a single value >= 2^31, and instead of doing the hex comparison, have the Java show that the value is cast to signed? Something like:
```
enum HowMany
{
  None,
  One,
  Many = 3,
  Lots = 0x80000000
}
```
In the Java:
```
// Enum values greater than INT_MAX will cast to their signed equivalents.
assert HowMany.Lots == Integer.MIN_VALUE;
```